### PR TITLE
basic pytorch support

### PR DESCRIPTION
**Pull request by ethantruelove**
Adds PyTorch support for getting metadata from models for DLHub servables.

### CODE REVIEW

**Comment by blaiszik** [comment]
Looks like tests are failing due to missing torch requirement. Can you add this and resend @ethantruelove?

**Comment by blaiszik** [comment]
Checks are still failing. You can check the details of the travis for more info. Looks like code formatting -- minor but important :)

File: dlhub_sdk/models/servables/pytorch.py [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+from dlhub_sdk.utils.types import compose_argument_block
+
+
+class TorchModel(BasePythonServableModel):
+    """Servable based on a Torch Model object.
+
+    Assumes that the model has been saved to a pt or a pth file"""
+
+    @classmethod
+    def create_model(cls, model_path, input_shape, output_shape):
```
Comment (WardLT) [inline dlhub_sdk/models/servables/pytorch.py]: Document the types and descriptions of each of the input arguments.

File: dlhub_sdk/models/servables/pytorch.py [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+        output = super(TorchModel, cls).create_model('predict')
```
Comment (WardLT) [inline dlhub_sdk/models/servables/pytorch.py]: Change 'predict' to '__call__'

**Comment by WardLT** [comment]
Thanks @ethantruelove !

**Pull request merged**